README.md

```diff
@@ -66,14 +66,15 @@ By definition, members work on their respective squad, although they are free to
 | Squad Name                    | Members                                     | Availability | Public slack channel |
 |-------------------------------|---------------------------------------------| ------------ | -------------------- |
 | Booking                       | Witold                                      |    1/1       | |
-| Condition Management          | Catarina, João, Dan S., Diego                      |    4/4       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) | 
```

Comment:
Where did @AnnKatF and @NSMyself end up, now that they're not in the Condition Managment squad anymore but have not migrated to Monitor either like Dan and @diegopetrucci did?

Reply:
You need to ask them because I honestly don't know.

Maybe @AnnKatF can shed some light.